3_javascript/javascript1/week1/startup-name-generator/scripts.js

```diff
@@ -0,0 +1,34 @@
+//startup-name-generator
+const firstWords = [
+  "Wecan",
+  "Undefined",
+  "Answers",
+  "Datany",
+  "Primeloop",
+  "Duany",
+  "Anyx",
+  "Virtan",
+  "Artyfy",
+  "Luvany",
+];
+
+const secondWords = [
+  "Onyx",
+  "Eligo",
+  "Haify",
+  "Motion",
+  "Chosen",
+  "Answered",
+  "Blitzly",
+  "Voxify",
+  "BlueSwifty",
+  "LevelLeap",
+];
+const randomFirstWord =
+  firstWords[Math.floor(Math.random() * firstWords.length)];
+const randomSecondWord =
+  secondWords[Math.floor(Math.random() * secondWords.length)];
+const startupName = `The startup: ${randomFirstWord} ${randomSecondWord} contains ${
```

Comment:
A little tiny comment here would be:

Instead of repeating the `Math.floor(Math.random() * firstWords.length` logic
You could avoid repetition by putting this logic in a constant variable called `randomNumber`. 
This will make the code more clean and easy to read.

Otherwise, really great job.

Reply:
Feedback Modified thanks @Elisha2605